android/app/src/main/java/com/reactnativebase/MainActivity.java

```diff
@@ -1,12 +1,23 @@
 package com.reactnativebase;
 
+import android.os.Bundle; // here
```

Comment:
what are the `// here` comments? I would delete them from here and line 9

Reply:
Copy/paste issue 😁